devices/devs.c

```diff
@@ -0,0 +1,153 @@
+/*
+ * Phoenix-RTOS
+ *
+ * phoenix-rtos-loader
+ *
+ * Device Interface
+ *
+ * Copyright 2021 Phoenix Systems
+ * Author: Hubert Buczynski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include "devs.h"
+#include "../errors.h"
+
+#define DEV_MAX_NB            8
+#define DEV_MAX_INSTANCE_NB   16
+
+/* dev_driver_t go to .bss section where is not zeroed
```

Comment:
"which", but the correct solution is to zero the `bss` - which can easily be added to `_start.c` function as a C code now.
This would make the `active` variable unneeded (and other istances of initializing variables to `0` not needed also).

Reply:
BSS is zeroed. `active` variable has been removed.